strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/artifact/coordinates/ArtifactLayout.java

```diff
@@ -0,0 +1,16 @@
+package org.carlspring.strongbox.artifact.coordinates;
+
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+@Target(TYPE)
+@Retention(RUNTIME)
+public @interface ArtifactLayout
```

Comment:
@carlspring please note that I this annotation satisfies your request regarding layout keywords which can be used in AQL

Reply:
Thank you very much!

@fuss86 , @steve-todorov : You might also find this useful someday. :)